code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -158,6 +158,12 @@
 /obj/item/ammo_box/magazine/internal/shot/toy/crossbow
 	max_ammo = 5
 
+//crossbow
+/obj/item/ammo_box/magazine/internal/shot/crossbow
+	ammo_type = /obj/item/ammo_casing/caseless/crossbowbolt && /obj/item/ammo_casing/apfsdsbolt

```

Comment:
Again, I don't consider this instance to be constant folding, the result of the operation is just constant.

This operation happens once, so its really just evaluating the operation, and nothing more.